README.md

```diff
@@ -72,4 +73,4 @@ now you can watch the ci build your awesome feature and publish your draft relea
 
 ## Minimum supported rust compiler
 
-This project's MSRV is `rustc 1.56`
```

Comment:
why the extra line

Reply:
Was autoadded by Github's integrated markdown editor (it's probably part of the MD spec to end files in a newline)
anyway, i got rid of it